.rubocop.yml

```diff
@@ -12,7 +12,7 @@ Metrics/BlockNesting:
 
 # TODO: this need some non-trivial refactoring...
 Metrics/ClassLength:
-  Max: 470
+  Max: 480
```

Comment:
IMHO, this is a symptom of refactoring needed. Not a big deal yet, but I suggest to add a FIXME into *migration_repos_workflow.rb* to indicate the class is already too long and requires refactoring.